uint16 id
std_msgs/Float64MultiArray goal